### ISSUE TITLE
a11y: `accTitle` and `accDescr` tags don't work in `block-beta` diagrams

### ISSUE THREAD

**Issue by lnihlen** [mermaid-js/mermaid#6524]
### Description

When I try to add `accTitle:` or `accDescr:` tags to an otherwise valid `block-beta` diagram, I get the following error:

```
Error: Error: Parse error on line 2:
block-beta    accTitle:    column
--------------^
Expecting 'SPACE_BLOCK', 'COLUMNS', 'id-block', 'block', 'NODE_ID', 'classDef', 'class', 'style', got 'acc_title'
```

It looks like the parser hasn't yet been configured to support these tags. I'd like every image on my site to have, at minimum, an accessible title and description.

Thanks!

### Steps to reproduce

When I try to add accessibility data to a `block-beta` diagram, the diagram fails to render.

### Screenshots

n/a

### Code Sample

```text
block-beta
  accTitle: Dependencies
  columns 3
  space Catalog[("Catalog")] space
  space:3
  LSP space Interpreter
  LSP --> Catalog
  LSP --> Interpreter
  Interpreter --> Catalog
```

### Setup

- Mermaid version: 11.6
- Browser and Version: Chrome 135.0.7049.96 (Official Build) (arm64) 

### Suggested Solutions

Extend the parser to support these tokens.

**Comment by lnihlen** [comment]
Friendly ping on this, thanks!
